gpiozero/pins/native.py

```diff
@@ -72,19 +76,17 @@ def close(self):
         self.mem.close()
         os.close(self.fd)
 
-    def peripheral_base(self):
+    def peripheral_base(self, rev):
```

Comment:
Perhaps this parameter should be renamed `cpu` to avoid confusion with the model revision codes?

Reply:
Yup, that's a good idea.